src/interface/src/app/map/map.component.scss

```diff
@@ -213,3 +213,26 @@
   pointer-events: auto;
   width: fit-content;
 }
+
+.config-summary {
+  display: none;
+}
+
+@media print {
+
+  .config-summary {
```

Comment:
Would it be possible to make this smaller to fit in more of the map?

Reply:
We don't know how much space the summary can take, it depends a lot on the layers and how many maps you are showing. Also if we make the map bigger Im not sure we'll have the data for the layers on the additional area. 
I think we can improve on this later if we need to but I rather not spend a lot of cycles on this now, what do you think?